es6-shim.js

```diff
@@ -205,7 +205,15 @@
     return result;
   };
 
+  var bindContext = function (func, context) {
+    return function () {
+      return func.apply(context, arguments);
+    };
+  };
+
   var ES = {
+    Call: bindContext(Function.prototype.call, Function.prototype.apply),
```

Comment:
I believe so. "The [[[Call]] internal method](https://people.mozilla.org/~jorendorff/es6-draft.html#sec-built-in-function-objects-call-thisargument-argumentslist) for a built-in function object F is called with parameters thisArgument and argumentsList."